tensorflow/examples/android/src/org/tensorflow/demo/CameraActivity.java

```diff
@@ -62,6 +87,103 @@ protected void onCreate(final Bundle savedInstanceState) {
     }
   }
 
+  /*
+ * Callback for android.hardware.Camera API
+ */
+  @Override
+  public void onPreviewFrame(final byte[] bytes, final Camera camera) {
+    if (computing) {
+      return;
+    }
+    computing = true;
+
+
+    Camera.Size previewSize = camera.getParameters().getPreviewSize();
+    yuvBytes[0]=bytes;
+    try {
+      // Initialize the storage bitmaps once when the resolution is known.
+      if (previewSize.width!=0 ||  previewSize.height!=0) {
+        previewHeight=previewSize.height;
+        previewWidth=previewSize.width;
+        rgbBytes = new int[previewWidth * previewHeight];
+        onPreviewSizeChosen(new Size(previewSize.width, previewSize.height), 90);
+        ImageUtils.convertYUV420SPToARGB8888(bytes, rgbBytes, previewWidth, previewHeight, false);
+
+      }
+
+    } catch (final Exception e) {
+      LOGGER.e(e, "Exception!");
+      return;
+    }
+
+    postInferenceCallback = new Runnable() {
+      @Override
+      public void run() {
+        camera.addCallbackBuffer(bytes);
+      }
+    };
+
+    processImageRGBbytes(new byte[bytes.length * 2/3]);
```

Comment:
that is a good question, in the original PR from @BrianOn99, used that value, I am not sure why ?